modules/osdk-project-file.adoc

```diff
@@ -58,6 +64,23 @@ resources:
 version: 3
 ----
 endif::[]
+ifdef::java[]
+----
+domain: example.com
```

Comment:
Can you please check the content of the file once?

I can see below when I scaffolded the project.

```
domain: example.com
layout:
- quarkus.javaoperatorsdk.io/v1-alpha
projectName: memcached-operator
version: "3"
```